src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryHolder.java

```diff
@@ -82,12 +85,17 @@ private HashMap<String, Object> getAllAsMap()
             Instant instant = Instant.ofEpochMilli(this.compactedAt);
             LocalDateTime ldt = LocalDateTime.ofInstant(instant, ZoneId.systemDefault());
             compaction.put("compacted_at", ldt.toString());
-            compaction.put("bytes_in", this.bytesIn);
-            compaction.put("bytes_out", this.bytesOut);
+            compaction.put("bytes_in", format(this.bytesIn, humanReadable));
+            compaction.put("bytes_out", format(this.bytesOut, humanReadable));
             compaction.put("rows_merged", this.rowMerged);
             compaction.put("compaction_properties", this.compactionProperties);
             return compaction;
         }
+
+        private String format(long bytes, boolean humanReadable)
```

Comment:
there are there methods like this if you grep the body of this method. Can not we create e.g. a static method in StatsHolder? Feel free to find better place to put it. For `example org.apache.cassandra.io.utils.FileUtils` is not bad either.

Reply:
I agree it should be static and should be given we take `humanReadable` as an argument. Not sure I love it in `FileUtils` unless we moved the logic into `stringifyFileSize` (e.g. pass `humanReadable` to it)

Reply:
Another alternative is don't have it as a function, make the call inline. If we do make it a function we should find other places where the logic is the same and replace it. otherwise we have a utility method only used in one place.

Reply:
I would generally prefer to put the logic into the stringifyFileSize and pass `humanReadable` to it because the TableStatsHolder also has its private format() method. Additionally, it would benefit other nodetool commands to have the -H options if yet has(I plan to vet all other nodetool commands separately later).